seeq/addons/correlation/_seeq_worksheet_writer.py

```diff
@@ -214,7 +215,18 @@ def worksheet_corrs_and_time_shifts(signal_pairs_ids: list, workbook_id: str,
                             condition_id,
                             overwrite,
                             api_client)
-    return "/".join([spy.client.host.replace('/api', ""), 'workbook', workbook_id, 'worksheet', existing_worksheet.id])
+
+    seeq_url_ = get_seeq_url()
+    
+    if seeq_url_ is None:
+        try:
+            with open('.urlfile/seeq_server_url.pkl', 'rb') as file_:
+                seeq_url_ = pickle.load(file_)
+        except FileNotFoundError:
+            raise FileNotFoundError(
```

Comment:
minor: This error message is not accurate. The issue here is that the file could not be found. While the root cause of that could be that the Seeq URL was not found before, there can be a number of other things that can cause this. 

I would suggest using

```python
except FileNotFoundError as e:
	raise(str(e))
```
Raising the actual error would be more informative since you could see the absolute path that is trying to reach.